src/java.desktop/macosx/classes/sun/lwawt/LWWindowPeer.java

```diff
@@ -491,12 +491,6 @@ public void updateIconImages() {
         getPlatformWindow().updateIconImages();
     }
 
-    @Override
-    public void setBackground(final Color c) {
-        super.setBackground(c);
-        updateOpaque();
```

Comment:
This code was added in the JDK-8033786, when it was implemented it was possible to call this method from the Window.setBackground() and from the Window.getPeer().setBackground().

Since the Window.setBackground() is always followed by the peer.setOpaque()->updateOpaque(), and now it is not possible to use the peer directly this method became redundant.

Deleted it in this fix since it generates a bunch of memory garbage.